src/js/components/campaignpage/CampaignSplashCard.jsx

```diff
@@ -33,15 +33,15 @@ const CampaignSplashCard = ({ branding, shareMeta, pdf, campaignPage, donationVa
 
 	const donationDisplay = <b>{donationValue ? <Counter amount={donationValue} minimumFractionDigits={2} preserveSize /> : "money"}</b>;
 
+	let [x, setX] = useState(undefined)
+	if(!x) {donationValue.value = 1340236.12; setX(donationValue);}
```

Comment:
Hello @LewisFramps - Could you add a comment to doc the logic behind this value? (which I expect we'll have to update periodically) Thanks

Reply:
that shouldn't have been left in, my bad!